### PR TITLE
Prometheus auto service discovery

### PR DESCRIPTION
**Pull request by maulindesai**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [X] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [X] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [X] Commands include options for Linux, MacOS, and Windows within codetabs
- [X] New file and folder names are globally unique
- [X] Page references use shortcodes instead of markdown or URL links
- [X] Images use HTML style and have alternative text
- [X] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

- Updated prometheus.md with instructions for setting up Prometheus auto service discovery for Dapr and sidecar targets.

## Issue reference
#4234

### CODE REVIEW

File: daprdocs/content/en/operations/observability/metrics/prometheus.md [hunk 1]
```diff
@@ -110,6 +205,22 @@ dapr-prom-prometheus-pushgateway-688665d597-h4xx2   1/1     Running   0
...
 
+### Access the Prometheus Dashboard
+
+To view the Prometheus dashboard and check service discovery:
+
+```bash
+kubectl port-forward svc/dapr-prom-prometheus-server 9090:80 -n dapr-monitoring
+```
+
+Open a browser and visit `http://localhost:9090`. Navigate to `Status -> Service Discovery` to verify that the Dapr targets are discovered correctly.
```
Comment (hhunter-ms) [inline daprdocs/content/en/operations/observability/metrics/prometheus.md]:
```suggestion
Open a browser and visit `http://localhost:9090`. Navigate to **Status** > **Service Discovery** to verify that the Dapr targets are discovered correctly.
```

**Review by hhunter-ms — CHANGES_REQUESTED** [review]
quick formatting fix to keep consistent w other docs. otherwise lgtm!

**Review by hhunter-ms — APPROVED** [review]

**Review by msfussell — APPROVED** [review]
LGTM

**Comment by marcduiker** [comment]
@holopin-bot @maulindesai Thank you for your contribution!

**Pull request merged**